common-theme/assets/styles/04-components/objectives.scss

```diff
@@ -3,7 +3,7 @@
 .c-objectives {
   margin-bottom: var(--theme-spacing--gutter);
   &__title {
-    margin-bottom: 0;
+    margin-bottom: .75em;
```

Comment:
please don't add ems, use the theme spacing.

the spacing can be found in spacing.scss